auth/src/main/java/org/aerogear/mobile/auth/configuration/KeycloakConfiguration.java

```diff
@@ -36,7 +38,9 @@
     public KeycloakConfiguration(final ServiceConfiguration configuration) {
         nonNull(configuration, "configuration");
 
-        this.serverUrl = nonEmpty(configuration.getProperty(SERVER_URL_NAME), SERVER_URL_NAME);
+        this.authServerUrl = nonEmpty(configuration.getProperty(AUTH_SERVER_URL_NAME),
```

Comment:
We should always use the url field in the config and not rely on the auth-server-url

Reply:
@TommyJ1994 Sorry I didn't make it clear in the JIRA. The "auth-server-url" field should not be used anywhere at all. Only the `url` field should be used.

Reply:
To verify JWT tokens (`iss` field), you need the actual Keycloak URL, not the Proxy/DMZ one as it will fail, so the `auth-server-url` is only used for token verification, the `url` field is used for everything else. That make sense?

Reply:
ah, right. OK, at long as it's not used for establish the communication with the server, then I think we are fine.

@TommyJ1994 can you add a comment to the code just to make it clear that the `auth-server-url` should only be used for token verification?

Reply:
@wei-lee Yeah, sure.